rust/scx_utils/src/compat.rs

```diff
@@ -17,6 +17,8 @@ lazy_static::lazy_static! {
     read_enum("scx_ops_flags", "SCX_OPS_SWITCH_PARTIAL").unwrap_or(0);
 }
 
+pub const SCX_BUILD_ID: &'static str = env!("VERGEN_GIT_DESCRIBE");
```

Comment:
Note that `VERGEN_GIT_DESCRIBE` can be confusing as it will show the scx repo version, not the scheduler version in `Cargo.toml`. They're not the same numbers. `resctl-demo` needs almost exactly the same thing and maybe we can just copy what it's doing? https://github.com/facebookexperimental/resctl-demo/blob/main/rd-util/src/lib.rs#L106